README.md

```diff
@@ -1,54 +1,68 @@
-# PipeRider
-
+[![ci-tests](https://github.com/infuseai/piperider-cli/actions/workflows/tests.yaml/badge.svg)](https://github.com/infuseai/piperider-cli/actions/workflows/tests.yaml/badge.svg)
+[![release](https://img.shields.io/github/release/infuseAI/piperider-cli/all.svg?style=flat-square)](https://github.com/infuseAI/piperider-cli/releases)
+[![InfuseAI Discord Invite](https://img.shields.io/discord/664381609771925514?color=%237289DA&label=chat&logo=discord&logoColor=white)](https://discord.com/invite/5zb2aK9KBV)
+
+<p align="center">
+  <a href="https://piperider.io">
+    <img  src="/.github/images/logo.svg"
+      width="284" border="0" alt="PipeRider">
+  </a>
+</p>
+
+# Automates data quality management
 For data practitioner who wants to trust their data by:  
 - Defining the shape of data, so they can **make sure the data will be expected in the future** 
 - Save time on debugging data,**easily discover & discuss any data problems from upstream**
 - Socially **collaborate across teams on the dataset through data catalog and data insights**
 
-# Install
-
-```bash
-$ pip install piperider-cli
-```
-
-### Install with connectors
+# Key Features
+## Instant quality assessment in html
+Check [single run view](/images/piperider_single_run.png)
 
-By default, we support the built-in sqlite connector, there are extra connectors:
+## Inject tests to key metrics
+Refer to [User defined test functions](https://github.com/InfuseAI/piperider-cli/blob/main/docs/user-defined-test-function.md)
 
-* snowflake
-* postgres
+## Compare different test assessments
+Check [comparison view](/images/piperider_comparison_view.png)
 
-Install with postgres connector:
+# Get started
+## Install PipeRider
 
-```
-pip install piperider-cli[postgres]
+```bash
+$ pip install piperider-cli
```

Comment:
Could change to `pip install piperider` directly.

Reply:
```suggestion
$ pip install piperider
```